ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -239,237 +238,234 @@ This program is distributed under the Artistic License.
 
 =cut
 
-
-my $Podroot;
-
-my %Pages = ();                 # associative array used to find the location
-                                #   of pages referenced by L<> links.
+sub new {
+    my $class = shift;
+    return bless {}, $class;
+}
 
 sub pod2html {
     local(@ARGV) = @_;
     local $_;
 
-    my $globals = init_globals();
+    my $self = Pod::Html->new();
```

Comment:
Could be written `__PACKAGE__->new`

Reply:
The way it's written is more self-documenting, so I think we should leave it this way for now.